src/unity/python/turicreate/toolkits/one_shot_object_detector/one_shot_object_detector.py

```diff
@@ -73,11 +74,123 @@ def __init__(self, state):
+            names as the annotations and feature used for model training.
+            Additional columns are ignored.
+
+        annotations : str
+            Name of the column that contains the annotations for the images.
+
+        **kwargs : optional Object Detector evaluate parameters
+
+        Returns
+        -------
+        out : dict / SFrame
+            Output type depends on the option `output_type`.
+
+        See Also
+        --------
+        create, predict
+
+        Examples
+        --------
+        >>> results = model.evaluate(data, "annotations")
+        >>> print('mAP: {:.1%}'.format(results['mean_average_precision']))
+        mAP: 43.2%
+        """
+        _raise_error_if_column_exists(dataset, annotations, 'dataset', annotations)
+        new_dataset = dataset.rename({annotations : 'annotation'})
+        return self.__proxy__['detector'].evaluate(new_dataset, **kwargs)
+
+    def export_coreml(self, filename, **kwargs):
+        """
+        Save the model in Core ML format. The Core ML model takes an image of
+        fixed size as input and produces two output arrays: `confidence` and
+        `coordinates`.
+
+        The first one, `confidence` is an `N`-by-`C` array, where `N` is the
+        number of instances predicted and `C` is the number of classes. The
+        number `N` is fixed and will include many low-confidence predictions.
+        The instances are not sorted by confidence, so the first one will
+        generally not have the highest confidence (unlike in `predict`). Also
+        unlike the `predict` function, the instances have not undergone
+        what is called `non-maximum suppression`, which means there could be
+        several instances close in location and size that have all discovered
+        the same object instance. Confidences do not need to sum to 1 over the
+        classes; any remaining probability is implied as confidence there is no
+        object instance present at all at the given coordinates. The classes
+        appear in the array alphabetically sorted.
+
+        The second array `coordinates` is of size `N`-by-4, where the first
+        dimension `N` again represents instances and corresponds to the
+        `confidence` array. The second dimension represents `x`, `y`, `width`,
+        `height`, in that order.  The values are represented in relative
+        coordinates, so (0.5, 0.5) represents the center of the image and (1,
+        1) the bottom right corner. You will need to multiply the relative
+        values with the original image size before you resized it to the fixed
+        input size to get pixel-value coordinates similar to `predict`.
+
+        See Also
+        --------
+        save
+
+        Parameters
+        ----------
+        filename : string
+            The path of the file where we want to save the Core ML model.
+
+        **kwargs : optional Object Detector export_coreml parameters
```

Comment:
Basically same comment as before: the object detector's export_coreml(...) currently only has three named parameters. Rather than having `kwargs`, I think it would be better to just reproduce the three parameters here.